ansible/Vagrantfile.Solaris10

```diff
@@ -2,12 +2,16 @@
 # vi: set ft=ruby :
 
 $script = <<SCRIPT
-sudo /opt/csw/bin/pkgutil -y -i python 
+sudo /opt/csw/bin/pkgutil -y -i python jdk8
```

Comment:
Hmmm - I'm assuming this is purely required as a prereq for the installer for the compiler? In which case can we separate out the `pkgutil jdk8` to be afer the `Install compiler if found on host system` section just to make it clear why we're installing it (or alternatively add a comment before this line, but it feels neater to group the related bits together

Reply:
Yep, JDK8 is to build the compiler :-)